Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Build final image
+FROM ubuntu:xenial
+
+# Import code, install code dependencies
+WORKDIR /srv
+ADD . /srv
+
+# System dependencies
+RUN apt-get update && apt-get install --yes python3-pip
+
+# Python dependencies
+ENV LANG C.UTF-8
+RUN pip3 install --upgrade pip && pip3 install gunicorn -r requirements.txt
+
+# Set git commit ID
+ARG commit_id
```

Comment:
After writing it this way, I now think it's better to do simply:

```
ARG COMMIT_ID=""
```

Firstly, I discovered there's no need to declare the argument as a proper environment variable - it works just the same without. Secondly, if we give it a default value then this repo can be build without the argument, e.g. for auto-building with hub.docker.com.

Reply:
I have updated this now